packages/cozy-viewer/package.json

```diff
@@ -52,7 +52,8 @@
     "hammerjs": "^2.0.8",
     "lodash": "^4.17.21",
     "react-markdown": "^4.0.8",
-    "react-pdf": "^5.7.2"
+    "react-pdf": "^5.7.2",
+    "react-router-dom": "^7.9.6"
```

Comment:
Oh I forgot:
- add it in devDeps ("react-router-dom": "^6.14.2") and in peerDeps ("react-router-dom": ">=6.14.2") instead of adding it here in deps to avoid multiple version of react-router-dom
- in this case, the commit become a breaking change

Reply:
You need to add "BREAKING CHANGE: explanation" to the commit message before merging it so it does a major version. 

Something like `BREAKING CHANGE: You need to add "react-router-dom" >= 6.14.2 as dep in your app`. You won't need to do anything when updating cozy-viewer in cozy-drive because it is already here but with this we keep a clean versioning.